chart/values.yaml

```diff
@@ -19,3 +19,29 @@ nats:
   logging:
     debug: false
     trace: false
+
+etcd:
+  ## Kubernetes Cluster Domain
+  ##
+  clusterDomain: cluster.local
+  ## TLS authentication for client-to-server communications
+  ## ref: https://etcd.io/docs/current/op-guide/security/
+  ##
+  client:
+    secureTransport: false
```

Comment:
FYI for hetzner k8s clusters, all intra-cluster traffic is encrypted with wireguard so this should be safe.

Reply:
Users will not be able to do it to start with because the code about to be checked in does not support secure transport. Both the code and the yaml need to be changed in tandem in the future.